content/lessons/chapter-2/scripting-2.tsx

```diff
@@ -125,7 +125,11 @@ const config: EditorConfig = {
 export default function Scripting2({ lang }) {
   const t = useTranslations(lang)
   const { currentLanguage } = useDataContext()
-  const [language, setLanguage] = useState(getLanguageString(currentLanguage))
+  const [language, setLanguage] = useState(
+    currentLanguage
+      ? getLanguageString(currentLanguage)
+      : config.defaultLanguage
+  )
```

Comment:
I ended up going the route of not immediately setting a default `currentLanguage` @daviroo do you think there is a more elegant way of solving this default to Javasscript issue?